mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/clients/enhetsregister/EregVirksomhetDto.kt

```diff
@@ -0,0 +1,26 @@
+package no.nav.mulighetsrommet.api.clients.enhetsregister
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class EregVirksomhetDto(
```

Comment:
Klarer vi å gjøre disse modellene interne innad i `enhetsregister`-modulen og heller mappe de om til f.eks. `VirksomhetDto`, eller en ny modell som vi ønsker for virksomheter innad i vårt domene?

Vi har ikke vært så nøye på å gjøre dette frem til nå, men det vil bidra til å holde tjenesten ryddigere etter hvert som den og antall eksterne avhengigheter vokser. Kan evt se på det sammen :+1:

Reply:
Ja, det er planen og ønske mitt også. I første omgang nå tenkte jeg bare å få sjekket at vi faktisk får kontakt med ereg og får data tilbake fra dem.